src/_wcs/components-template/Component-Template/new-component.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import './new-component.st.css';
+
+export const NewComponent = () => (
+    <div className="root">I am a NewComponent</div>
```

Comment:
@omerken I would do something a bit more descriptive here:
Since users that try out this project will now skip over the configuration of a template they will miss this feature completely. 
I would improve this template by maybe wrapping it with another paragraph that tells the user "hey, this is a custom component template. you can configure your own! [learn more]"

Reply:
That's a good idea, but this pr is only relevant for this temporary Example Project. I'm using it for demos, but this is not going to be the actual example project.

@mikawix - See Marton's comment regarding the template of the actual demo project

Reply:
@omerken AFAIK we're directing users to clone this repo in the "getting started" guide